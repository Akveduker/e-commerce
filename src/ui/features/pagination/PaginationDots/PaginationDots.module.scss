.dots {
    margin-left: 8px;
}
