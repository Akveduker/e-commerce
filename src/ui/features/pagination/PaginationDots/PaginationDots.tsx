import React from 'react';
import s from './PaginationDots.module.scss'
const PaginationDots = () => {
    return (
        <span className={s['dots']}>
            ...
        </span>
    );
};

export default PaginationDots;