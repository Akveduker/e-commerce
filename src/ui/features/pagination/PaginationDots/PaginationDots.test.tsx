import { render } from "@testing-library/react"
import PaginationDots from "./PaginationDots"

describe('PaginationDots', () => {
    test('Снапшот', () => {
        const { container } = render(<PaginationDots />)
        expect(container).toMatchSnapshot()
    })
})