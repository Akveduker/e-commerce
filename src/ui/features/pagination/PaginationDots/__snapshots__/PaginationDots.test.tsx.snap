// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationDots Снапшот 1`] = `
<div>
  <span
    class="dots"
  >
    ...
  </span>
</div>
`;
